internal/engine/actions/alert/pull_request_comment/pull_request_comment.go

```diff
@@ -0,0 +1,262 @@
+// SPDX-FileCopyrightText: Copyright 2024 The Minder Authors
+// SPDX-License-Identifier: Apache-2.0
+
+// Package pull_request_comment provides necessary interfaces and implementations for
+// processing pull request comment alerts.
+package pull_request_comment
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"math"
+	"time"
+
+	"github.com/google/go-github/v63/github"
+	"github.com/rs/zerolog"
+	"google.golang.org/protobuf/reflect/protoreflect"
+
+	"github.com/mindersec/minder/internal/db"
+	enginerr "github.com/mindersec/minder/internal/engine/errors"
+	"github.com/mindersec/minder/internal/engine/interfaces"
+	pbinternal "github.com/mindersec/minder/internal/proto"
+	pb "github.com/mindersec/minder/pkg/api/protobuf/go/minder/v1"
+	"github.com/mindersec/minder/pkg/profiles/models"
+	provifv1 "github.com/mindersec/minder/pkg/providers/v1"
+)
+
+const (
+	// AlertType is the type of the pull request comment alert engine
+	AlertType = "pull_request_comment"
+)
+
+// Alert is the structure backing the noop alert
+type Alert struct {
+	actionType interfaces.ActionType
+	gh         provifv1.GitHub
+	reviewCfg  *pb.RuleType_Definition_Alert_AlertTypePRComment
+	setting    models.ActionOpt
+}
+
+type paramsPR struct {
+	Owner      string
+	Repo       string
+	CommitSha  string
+	Number     int
+	Metadata   *alertMetadata
+	prevStatus *db.ListRuleEvaluationsByProfileIdRow
+}
```

Comment:
Right, so... If I understand correctly, this are the parameters to create a PR regardless of the underlying driver, right?

Reply:
That's right